floodfill_selection/volume_selection.py

```diff
@@ -150,7 +149,7 @@ def __init__(self, keys='interactive'):
         new_pos = np.transpose(self.vol_data)
 
         # TODO: replace the min&max threshold with real settings in Glue UI
-        self.pos_data = np.argwhere(new_pos >= np.min(self.vol_data))  # get voxel positions
+        self.pos_data = np.indices(self.vol_data.shape).reshape(3,-1).transpose()
```

Comment:
i think that ``indices`` will be faster, but I think the result will be the same.